plugins/module_utils/ece.py

```diff
@@ -115,8 +115,12 @@ def get_deployment_kibana_info(self,deployment_name):
       deployment_objects = self.send_api_request(endpoint, 'GET')
       for deployment in deployment_objects['deployments']:
         if str(deployment['name']).upper() == str(deployment_name).upper():
-          endpoint  = 'deployments/' + deployment['id'] + '/kibana/main-kibana'
-          deployment_object = self.send_api_request(endpoint, 'GET')
-          break
+          try:
+            endpoint  = 'deployments/' + deployment['id'] + '/kibana/main-kibana'
+            deployment_object = self.send_api_request(endpoint, 'GET')
```

Comment:
I was looking for a quick fix because this is holding up work of delivery. There is probably a way to find the resource name for kibana, but it would require parsing through a lot of JSON.  I can look into it, but I would like to get this out the door.

Reply:
If this is holding up business, I'd rather you push a dev version of this collection to Hub to solve the immediate business need and we can keep the updates in this PR and the repo as best practice. Let me know if you need assistance with pushing/pulling a '-dev' version of the collection from hub with tower.